docs/src/pages/components/tables/tables.md

```diff
@@ -19,7 +19,7 @@ When including tools, they should be placed directly above or below the table.
 
 ## Structure
 
-A data table contains a header row at the top that lists column names, followed by rows for data.
+A data table can contain one or more header rows at the top, followed by rows for data. The header row lists column names of the table.
```

Comment:
What about going for a different approach, the HTML table is very flexible, I'm not sure we cal list of the combinations. I think that the issue's author comes from either a lack of knowledge of the composition capability of native HTML table elements or from missing information, that Material-UI Table components directly wraps native table elements.

Reply:
For benchmark, ag-grid has a section about grouping: https://ag-grid.com/javascript-grid-grouping-headers/.

Reply:
Ohh, I thought this was just a documentation issue. Are you asking me to implement a feature to group column headers? It might be over my head, but I would love to try it.

Reply:
It's a documentation only issue. I would like to explore the alternatives we have to document it.